english/eating-walls/eating-marshmallows.md

```diff
@@ -6,8 +6,7 @@ this story happened to provide you. Shoveling the fluffy, whipped sugar into you
 you begin to feel a sense of satisfaction.
 
 Sitting down next to the partially eaten marshmallow wall, your stomach begins to grumble, churn,
-and babble. Hoping to walk off the uneasy feeling, you stand up. Upon standing up, your stomach
-quiets and you entertain how to spend the rest of the day.
+and babble. Hoping to walk off the uneasy feeling, you stand up. You quickly realize that as you digest the marshmallow, the gooey, sugary treat begins eating away at you from the inside! 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.